doc/en/components/grids/_shared/excel-style-filtering.md

```diff
@@ -41,7 +41,7 @@ To turn on the `{ComponentName}` component's Excel-style filtering, two inputs s
 
 <!-- React -->
 ```tsx
-<{ComponentSelector} data={nwindData} autoGenerate="true" allowFiltering="true" filterMode={FilterMode.ExcelStyleFilter}>
+<{ComponentSelector} data={nwindData} autoGenerate={true} allowFiltering={true} filterMode={FilterMode.ExcelStyleFilter}>
```

Comment:
`filterMode={FilterMode.ExcelStyleFilter}` no longer working because it is exported as type and not enum.
You can go like this: ` filterMode="excelStyleFilter"`